### PR TITLE
docs: add Web Worker guide

### PR DESCRIPTION
**Pull request by filipesilva**
Followup to https://github.com/angular/angular-cli/pull/13700

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -414,6 +414,7 @@
 
 /packages/compiler-cli/src/ngtools/**                           @angular/tools-cli @angular/framework-global-approvers
 /aio/content/guide/ivy.md                                       @angular/tools-cli @angular/framework-global-approvers @angular/framework-global-approvers-for-docs-only-changes
+/aio/content/guide/web-worker.md                                @angular/tools-cli @angular/framework-global-approvers @angular/framework-global-approvers-for-docs-only-changes
```
Comment (gkalpak) [inline .github/CODEOWNERS]: :100:

File: aio/content/guide/web-worker.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- add `src/app/app.worker.ts` with scaffolded code to receive messages:
+
+  ```typescript
+  addEventListener('message', ({ data }) => {
+    const response = `worker response to ${data}`;
+    postMessage(response);
+  });
+  ```
+
+- add scaffolded code to `src/app.component.ts` to use the worker:
```
Comment (gkalpak) [inline aio/content/guide/web-worker.md]: Did you mean `src/app/app.component.ts`?

File: aio/content/guide/web-worker.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  }
+  ```
+
+After the initial scaffolding, you will need to refactor your code to use the Web Worker by sending messages to and from.
+
+## Caveats
+
+There are two important things to keep in mind when using Web Workers in Angular projects:
+
+- Some environments or platforms, like `@angular/platform-server` used in [Server-side Rendering](guide/universal), don't support Web Workers. You have to provide a fallback to mechanism to perform the computations that the worker would perform to ensure your application will work in these environments.
```
Comment (gkalpak) [inline aio/content/guide/web-worker.md]: _fallback to mechanism to perform_ --> _fallback mechanism to perform_

**Review by gkalpak — APPROVED** [review]

File: aio/content/guide/web-worker.md (hunk 2 above)
Comment (filipesilva) [reply aio/content/guide/web-worker.md]: I did, yes. Thank you.

File: aio/content/guide/web-worker.md (hunk 3 above)
Comment (filipesilva) [reply aio/content/guide/web-worker.md]: Done

**Review by alexeagle — APPROVED** [review]

**Comment by filipesilva** [comment]
caretaker: I believe the current reviews should be all that are needed for this PR.

**Comment by jasonaden** [comment]
I'm going to take the action of labeling this `merge` from @alexeagle to mean he wants it merged.

**Pull request closed without merging**